etc/local_grapl/bin/upload_analyzer_local.sh

```diff
@@ -3,16 +3,16 @@ set -eu
 
 ANALYZER_UPLOAD_SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 LOCAL_GRAPL_DIR=$ANALYZER_UPLOAD_SCRIPT_PATH/../
-export AWS_ACCESS_KEY_ID=minioadmin 
-export AWS_SECRET_ACCESS_KEY=minioadmin 
+export AWS_ACCESS_KEY_ID="${S3_ACCESS_KEY_ID}"
```

Comment:
Again, has this been manually tested? Could you document "hey, run <some script> that will export S3_ACCESS_KEY_ID etc etc"?